store/mongo/tenant_datastore.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package mongo
+
+import (
+	"context"
+)
+
+type TenantStoreMongo struct {
+	db *DataStoreMongo
+}
+
+func NewTenantStoreMongo(db *DataStoreMongo) *TenantStoreMongo {
+	return &TenantStoreMongo{
+		db: db,
```

Comment:
didn;t you mean to set the db as AutoMigrate()'ed?

Reply:
Added fix in the next commit